src/OrchardCore.Modules/OrchardCore.Setup/Views/Setup/Index.cshtml

```diff
@@ -21,14 +21,14 @@
     if (options.Password.RequireNonAlphanumeric)
     {
         passwordOptions.AppendHtml(separator).AppendHtml(T["one non-alphanumeric"]);
-        separator = T[", "];
+        separator = ", ";
```

Comment:
What if RTL needs the space on the left.

Reply:
I think it's fine but I need to check this